quickstart.rst

```diff
@@ -0,0 +1,57 @@
+.. _quickstart:
+
+Quick Start
+===========
+
+The easiest way to get started with Multinet is to go to https://multinet.app,
+and begin experimenting.
+
+To build a local development environment, you will need to *install
+dependencies*, *launch a database*, *build and run the server*, and *build and
+run the client*.  Follow these steps:
+
+Install Dependencies
+--------------------
+
+These projects are built using the following tools; ensure that you have them
+installed:
+- Docker
+- Python 3.7
+- Pipenv
+- Node
+- Yarn
+
+Build and Run ``multinet-server``
+---------------------------------
+
+1. Clone the repository: ``git clone https://github.com/multinet-app/multinet-server``.
+2. Move into the repository: ``cd multinet-server``.
+3. Copy the ``.env.default`` file: ``cp .env.default .env`` (or symlink it: ``ln
+   -s .env.default .env``).
+4. Install the Python dependencies: ``pipenv install``.
+5. Activate the virtual environment: ``pipenv shell``.
+6. Run the server application in dev mode: ``pipenv run serve``.
+7. Point your web browser to http://localhost:5000 to ensure that the server is
+   working.
+
+Launch ArangoDB
+---------------
+
+1. Move into the server repository's ``devops`` directory: ``cd
+   multinet-server/devops``.
+2. Run the docker container for ArangoDB: ``docker-compose up``.
```

Comment:
This is currently incorrect. Are you planning to change this over soon?

Reply:
What's incorrect about it?

Reply:
The file doesn’t live in that folder, it’s at the root of the multinet-server repo

Reply:
Good catch. See 46c57f20038deb85ef7bf271d683331a21295486.